scripts/debug_course_publisher.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import time
+
+import rospy
+import tf2_ros
+from geometry_msgs.msg import TransformStamped
+
+
+def send_waypoint(name: str, x: float, y: float):
```

Comment:
I take it the real thing will linearize from gps.

Reply:
Yes it should, which is interesting since we will need to hook into robot_localization package

Reply:
Whys that? Just import the utils function we wrote

Reply:
Just gotta make sure it is the same as the package we use, can probably look at the src